wave_lang/kernel/ops/wave_ops.py

```diff
@@ -1390,6 +1400,7 @@ class AtomicOp(BinaryOpBase, ABC):
 
     elements_per_thread: Optional[Any] = None
     mapping: Optional[IndexMapping] = None
+    mapping_dynamic_vals: tuple[fx.Node, ...] = ()
```

Comment:
Unrelated to this PR exactly, but why does AtomicOp need to inherit from both BinaryOpBase and ABC, when BinaryOpBase has ABC in its inheritance chain?  Just curious.